src/com/carolinarollergirls/scoreboard/json/ScoreBoardJSONListener.java

```diff
@@ -280,6 +281,25 @@ private void processPosition(String path, Position p, boolean remove) {
 		updateMap.put(path + "." + Position.EVENT_SKATER, p.getSkater() == null ? null : p.getSkater().getId());
 		updateMap.put(path + "." + Position.EVENT_PENALTY_BOX, p.getPenaltyBox());
 	}
+	
+	private void processPenalties(Settings s) {
+		PenaltiesManager pm = new PenaltiesManager(s);
+
+        try {
+            PenaltiesDefinition penalties = pm.loadFromJSON();
+            for(PenaltyCode p : penalties.getPenalties()) {
+                
+                updateMap.put("PenaltyCode.("+p.getCode()+")", p);
```

Comment:
While you're changing all this, would you mind moving this under `ScoreBoard.`? That way a WS client will (eventually) be able to register `Scoreboard.` and get everything.